playbooks/templates/hashicorp-vault/docker-compose.yml.j2

```diff
@@ -0,0 +1,50 @@
+version: '3'
+
+services:
+  traefik:
+    # The latest official supported Traefik docker image
+    image: traefik:v2.8
+    container_name: traefik
+    # Enables the Traefik Dashboard and tells Traefik to listen to docker
+    # enable --log.level=INFO so we can see what Traefik is doing in the log files
+    ports:
+      # Exposes port 80 for incomming web requests
+      - "80:80"
+      - "443:443"
```

Comment:
Can we change these? I see they are defines in the traefik config file as well.

If we can make them something odd, like 7080, that would allow portal operators to use one of their operational servers (or all of them) as deployment servers as well. If I am getting this right, right now it's not possible to use these playbooks from a running server because of this port collission.

Reply:
Created a FU: #384